### ISSUE TITLE
Idea: Windows installers

### ISSUE THREAD

**Issue by kirbyfan64** [alda-lang/alda-core#32]
I would be happy to contribute them! :D

**Comment by abhi18av** [comment]
@kirbyfan64 , do you plan to use [Chocolatey](https://chocolatey.org/) ? That'd be really helpful, though I don't know if it works more like `brew` i.e. builds packages or `brew cask` i.e. installs binaries.

I've made a couple attempts with the `brew cask` feature - will complete it as soon as I get some extra time on my hands.

Thanks, btw.

**Comment by daveyarwood** [comment]
+1 to Chocolatey or something similar. With Alda being a command-line program, it would be nice to manage it via existing command-line-oriented package managers for each OS.

**Comment by kirbyfan64** [comment]
True, I forgot about Chocolatey. (I maintain the PyPy package... though it's a few versions behind...)
